Grizzlies-SpyDuh/Database/SpyDuh.sql

```diff
@@ -77,16 +77,18 @@ GO
 CREATE TABLE [Assignment] (
   [Id] int PRIMARY KEY identity,
   [Description] nvarchar(255) not null,
-  [UserService] int not null,
+  [AgencyId] int not null,
   [Fatal] bit not null,
-  [StartMissionDate] datetime not null,
-  [EndMissionDate] datetime 
+  [StartMissionDateTime] datetime not null,
+  [EndMissionDateTime] datetime 
 )
```

Comment:
Changed these columns to more specific names. No one had used assignment data yet, so shouldn't result in any errors